src/js/legend.js

```diff
@@ -258,4 +323,11 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
         return colorScale.range();
     };
 
+    floe.chartAuthoring.pieChart.legend.getDisplayValueFromTemplate = function(that, template, d) {
+        var legendPercentageDigits = that.options.legendOptions.legendPercentageDigits;
+        var percentage = floe.chartAuthoring.percentage.calculate(d.value, that.model.total.value).toFixed(legendPercentageDigits);
```

Comment:
floe.chartAuthoring.percentage.calculate() could return ```null``` in which case using ```toFixed()``` on a null value would throw javascript error.